data/json/monsters/mutant_mammal.json

```diff
@@ -364,5 +445,160 @@
       "SWARMS",
       "WARM"
     ]
+  },
+  {
+    "id": "mon_bigdog_greatestswiss",
+    "type": "MONSTER",
+    "name": { "str_sp": "Greatest Swiss" },
+    "description": "This dog is absolutely massive.  It would be big enough to pull a full-sized car.  It might even be ridable.  However, it does appear to be fiercely protective of the other dogs around it.",
+    "//": "For mutated dogs, trying to exaggerate what they were good at",
+    "default_faction": "dog",
+    "bodytype": "dog",
+    "categories": [ "WILDLIFE" ],
+    "species": [ "MAMMAL" ],
+    "volume": "400000 ml",
+    "weight": "400 kg",
+    "hp": 270,
+    "speed": 145,
```

Comment:
110 attack cost to 145 speed is very dangerous. I'd bump the speed down to 130 or the attack cost to match, and move most of their damage to scratch/bite attacks at higher cost (this is a general observation about all of them).